eZ/Publish/Core/Repository/RoleService.php

```diff
@@ -434,6 +825,32 @@ public function loadRole($id)
     }
 
     /**
+     * Loads a role draft for the given identifier.
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException if the authenticated user is not allowed to read this role
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the authenticated user is not allowed to read this role
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException if a role with the given name was not found
+     *
+     * @param string $identifier
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\RoleDraft
+     */
+    public function loadRoleDraftByIdentifier($identifier)
```

Comment:
@andrerom I guess it was added for completeness, and for detecting role edit conflicts.